### ISSUE TITLE
Unknown Error when try to add the integration 

### ISSUE THREAD

**Issue by Gitspecial** [bremor/bureau_of_meteorology#105]
This error originated from a custom integration.

Logger: custom_components.bureau_of_meteorology.config_flow
Source: custom_components/bureau_of_meteorology/PyBoM/collector.py:42
Integration: Bureau of Meteorology (documentation, issues)
First occurred: 12:37:39 PM (6 occurrences)
Last logged: 12:39:27 PM

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/bureau_of_meteorology/config_flow.py", line 49, in async_step_user
    await self.collector.async_update()
  File "/config/custom_components/bureau_of_meteorology/PyBoM/collector.py", line 95, in async_update
    await self.format_daily_forecast_data()
  File "/config/custom_components/bureau_of_meteorology/PyBoM/collector.py", line 42, in format_daily_forecast_data
    days = len(self.daily_forecasts_data["data"])
KeyError: 'data'

**Comment by Makin-Things** [comment]
Please provide the Lat/Lon you are using so I can try and replicate.

**Comment by Gitspecial** [comment]
27.568547,153.116494

**Comment by Makin-Things** [comment]
I believe this is a duplicate of #95 that I released a beta patch for yesterday (1.1.4b). Please test that.
You didn't say but I suspect you are not ticking the forecast box when configuring.

**Comment by Gitspecial** [comment]
sorry but where do I tick forecast box? I only get option of longitude and latitude, nothing else to select.

**Comment by Makin-Things** [comment]
Try hard refreshing your browser. 
Note the the first dialog only asks for coordinates and the rest of the settings are made on subsequent dialogs.

**Comment by Gitspecial** [comment]
Thanks, I just can't get past the first dialogue, getting Unknown Error. So the issue I raised originally still exist.

**Comment by Makin-Things** [comment]
Are you located in Brisbane? The coordinates you provided seem too indicate so, but the 27.568547 needs to have a - sign at the front to indicate southern hemisphere.
If that fixes it for you please comment, but don't close the issue as we will look at better error checking.

**Comment by Gitspecial** [comment]
Thanks, it did work after putting minus sign in front. Thanks a lot.

**Comment by Makin-Things** [comment]
Great. Thanks for the feedback. I'll work on getting the integration to provide a better error message.

**Comment by whatsupskip** [comment]
I kept receiving errors when trying to enter the GPS locations. I noticed that it seem to remove the negative sign from the latitude entry while I was typing the numbers. The longitude value seem to keep deleting the dot for the decimal place. I would also receive an error message stating that the longitude value was not a "float" value.

Eventually I managed to have it accept the values or at least near enough for it to work.

Thanks for all you efforts with developing this module.
